app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,16 @@
+@import "sass/core";
+
+// body{
+// 	background: url("dolores.jpg");
+// 	background-size: 100%;
+// }
+// .container{
+// 	background: white;
+// 	padding: 50px;
+// }
+
+.logo-18f{
+	width: 25px;
```

Comment:
is this standard indentiation for css? I usually see a space before `{` and 2 spaces for indentation - seems like that is also the preferred 18F style https://github.com/18F/frontend/blob/18f-pages-staging/pages/css-styleguide/format.md